bipolarbridges/companion-app
# Automated QA Pipeline for Android App (and Other CI Tweaks)

## Changes
Adds steps to the `build` action to build the android app and type check the mobile directory. Also splits with existing checks into smaller workflows.

## Requirements
N/A

## Subtasks / Progress
- [x] Add build job for Android
- ~~[ ] Add build job for iOS (currently blocked by GitHub Actions' lack of support for macOS-11 and Xcode 12.5)~~
- [x] Add lint job for mobile directory (errors to be fixed in #85)


## Testing

Check the build action to see logs of the android build

- [x] Automated

List of added tests:
- install mobile
- build android
- lint mobile
